toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/tools/test_xnvme_file.py

```diff
@@ -0,0 +1,21 @@
+from ..conftest import xnvme_parametrize
+
+
+@xnvme_parametrize(labels=["file"], opts=["be", "sync"])
+def test_copy_sync(cijoe, device, be_opts, cli_args):
```

Comment:
True, the unused ``cli_args`` is not pretty, however, it is provided since ``xnvme_parametrize()`` produces the list of ``(device, be_opts, cli_args)``, if the argument is removed from the test-func, then it fails.